test/style/function/composite_function.test.cpp

```diff
@@ -0,0 +1,46 @@
+#include <mbgl/test/util.hpp>
+#include <mbgl/test/stub_geometry_tile_feature.hpp>
+
+#include <mbgl/style/function/composite_function.hpp>
+
+using namespace mbgl;
+using namespace mbgl::style;
+
+using namespace std::string_literals;
+
+static StubGeometryTileFeature oneInteger {
+    PropertyMap {{ "property", uint64_t(1) }}
+};
+
+TEST(CompositeFunction, ZoomInterpolation) {
+    EXPECT_EQ(40.0f, CompositeFunction<float>("property", CompositeExponentialStops<float>({
+        {0.0f, {{uint64_t(1), 24.0f}}},
+        {1.5f, {{uint64_t(1), 36.0f}}},
+        {3.0f, {{uint64_t(1), 48.0f}}}
+    }), 0.0f)
+    .evaluate(2.0f, oneInteger, -1.0f)) << "Should interpolate between stops";
+    
+    EXPECT_EQ(0.0, CompositeFunction<float>("property", CompositeExponentialStops<float>({
```

Comment:
Shouldn't the expected result be `33.0`? Functions of all types are supposed to be flat before their first stop.